### ISSUE TITLE
API throws error from uwsgi cron job when encountering stale "running" job.

### ISSUE THREAD

**Issue by ryansanford** [scitran/api#111]
## Error
uwsgi_1          | [pid: 71|app: 0|req: 1/1] 192.168.99.1 () {34 vars in 399 bytes} [Wed Dec 30 19:37:47 2015] GET /api/jobs/next => generated 864 bytes in 35 msecs (HTTP/1.1 200) 3 headers in 112 bytes (2 switches on core 0)
uwsgi_1          | Traceback (most recent call last):
uwsgi_1          |   File "bin/api.wsgi", line 87, in job_creation_timer
uwsgi_1          |     job_creation(signum)
uwsgi_1          |   File "bin/api.wsgi", line 76, in job_creation
uwsgi_1          |     jobs.retry_job(config.db, j)
uwsgi_1          | NameError: global name 'jobs' is not defined
uwsgi_1          | Wed Dec 30 19:39:47 2015 - error managing signal 0 on worker 4

## How to reproduce
1. bootstrap new instance of API
2. Wait for API to create jobs from bootstrapped data.
3. Set job to running state
```
baseUrl="https://localhost:8443/api/jobs"
alias sCurl="curl -sk -H 'User-Agent: SciTran Drone Engine' -H 'X-SciTran-Auth: change-me'"
sCurl $baseUrl/next

```
4. Wait 100 seconds.

If this error happens on uwsgi startup rather than cron execution, all subsequent api requests will error with http 500 response codes until uwsgi is restarted enough times to reset all stale "running" jobs.

**Comment by kofalt** [comment]
If this was fixed in #113, close :)

**Comment by ryansanford** [comment]
It was @kofalt . Closing...
